src/ResourceManager/Compute/Commands.Compute/packages.config

```diff
@@ -4,6 +4,6 @@
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="17.3.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.KeyVault" version="2.3.0-preview" targetFramework="net452" />
```

Comment:
We need to remove these dependencies.  If these dependencies on Storage and KeyVault are needed, we should move this into captured common code.